x-pack/plugins/index_management/public/application/sections/home/enrich_policies_list/enrich_policies_list.tsx

```diff
@@ -79,7 +77,7 @@ const ListView: React.FunctionComponent<RouteComponentProps> = ({ history, locat
     return <ErrorState error={error} resendRequest={reloadPolicies} />;
   }
 
-  if (policies?.length === 0) {
+  if (capabilities.index_management.manageEnrich && policies?.length === 0) {
```

Comment:
With this change we lose the empty state if they dont have the right permissions, how about we remove the CTA in the empty state component instead?